### PR TITLE
 Docs(web, web-react, web-twig): Add Breadcrumbs demo #DS-981

### PR DESCRIPTION
**Pull request by pavelklibani**
## Description

Breadcrumbs demos are same in `web`, `web-react` and `web-twig`.

## Additional info

The demo in the `web` section has copied the code to appear the same as the demos in `web-react` and `web-twig`, where you can implement the `<Breadcrumbs />` in two different ways so we can run visual tests on all demos.

### Issue reference

[Demo: Breadcrumbs má ve všech technologiích stejná dema](https://jira.lmc.cz/browse/DS-891)

### CODE REVIEW

File: packages/web/src/scss/components/Breadcrumbs/index.html [hunk 1]
```diff
@@ -1,35 +1,77 @@
...
+        </svg>
+        <a href="#currentUrl" aria-current="page" class="link-secondary">Current page</a>
+      </li>
+    </ol>
+  </nav>
+</section>
+
+<!-- 
+  This code is a copy of the 'Default' section. 
+  We need a visually identical demo to Twig & React, where we have both the 'Default' and 'Custom' implementations.
```
Comment (literat) [inline packages/web/src/scss/components/Breadcrumbs/index.html:40]:
```suggestion
  We need a visually identical demo of the Twig & React, where we have both the 'Default' and 'Custom' implementations.
```

**Review by literat — APPROVED** [review]
LGTM 👍

File: packages/web/src/scss/components/Breadcrumbs/index.html (hunk 1 above)
Comment (pavelklibani) [reply packages/web/src/scss/components/Breadcrumbs/index.html]: fixed

File: packages/web-twig/src/Resources/components/Breadcrumbs/stories/BreadcrumbsCustom.twig [hunk 2]
```diff
@@ -1,23 +1,26 @@
...
-    <li class="d-none d-tablet-flex">
-        <Icon name="chevron-right" />
-        <Link href="#currentUrl" color="secondary" aria-current="page">Current page</Link>
-    </li>
-  </ol>
-</Breadcrumbs>
+<section class="docs-Section">
+  <h2 class="docs-Heading">Custom</h2>
+    <Breadcrumbs>
+    <ol>
```
Comment (crishpeen) [inline packages/web-twig/src/Resources/components/Breadcrumbs/stories/BreadcrumbsCustom.twig:4]: There should be more space as this is inside the Breadcrumbs. Please fix the indentation.

File: packages/web-react/src/components/Breadcrumbs/demo/index.tsx [hunk 3]
```diff
@@ -6,36 +6,18 @@ import ReactDOM from 'react-dom/client';
...
-          goBackTitle="Back"
-        />
-      </IconsProvider>
-    </DocsSection>
+    <IconsProvider value={icons}>
+      <DocsSection title="Default">
+        <BreadcrumbsDefault />
+      </DocsSection>
+      <DocsSection title="Custom">
+        <BreadcrumbsDefault />
```
[the remark this answers is not in the document]
Comment (pavelklibani) [reply packages/web-react/src/components/Breadcrumbs/demo/index.tsx:19]: yes, it should be  <BreadcrumbsCustom />, thanks

File: packages/web-twig/src/Resources/components/Breadcrumbs/stories/BreadcrumbsBasic.twig [hunk 4]
```diff
@@ -17,4 +17,7 @@
   },
 ] %}
 
-<Breadcrumbs items={{ items }} />
+<section class="docs-Section">
```
[the remark this answers is not in the document]
Comment (pavelklibani) [reply packages/web-twig/src/Resources/components/Breadcrumbs/stories/BreadcrumbsBasic.twig:20]: done

File: packages/web-twig/src/Resources/components/Breadcrumbs/stories/BreadcrumbsCustom.twig (hunk 2 above)
Comment (pavelklibani) [reply packages/web-twig/src/Resources/components/Breadcrumbs/stories/BreadcrumbsCustom.twig:4]: fixed

File: packages/web-twig/src/Resources/components/Breadcrumbs/Breadcrumbs.stories.twig [hunk 5]
```diff
@@ -3,9 +3,15 @@
 {% block content %}
 
 {# Breadcrumbs Basic #}
-{% include '@components/Breadcrumbs/stories/BreadcrumbsBasic.twig' %}
+<section class="docs-Section">
+  <h2 class="docs-Heading">Default</h2>
+  {% include '@components/Breadcrumbs/stories/BreadcrumbsBasic.twig' %}
```
Comment (crishpeen) [inline packages/web-twig/src/Resources/components/Breadcrumbs/Breadcrumbs.stories.twig:8]: please rename the file to BreadcrumbsDefault

File: packages/web-twig/src/Resources/components/Breadcrumbs/stories/BreadcrumbsBasic.twig [hunk 6]
```diff
@@ -17,4 +17,4 @@
   },
 ] %}
 
-<Breadcrumbs items={{ items }} />
+<Breadcrumbs items={{ items }} />
```
Comment (crishpeen) [inline packages/web-twig/src/Resources/components/Breadcrumbs/stories/BreadcrumbsBasic.twig:20]: an empty line at the bottom is missing?

**Review by crishpeen — APPROVED** [review]

File: packages/web-twig/src/Resources/components/Breadcrumbs/Breadcrumbs.stories.twig (hunk 5 above)
Comment (pavelklibani) [reply packages/web-twig/src/Resources/components/Breadcrumbs/Breadcrumbs.stories.twig]: renamed to BreadcrumbsDefault

File: packages/web-twig/src/Resources/components/Breadcrumbs/stories/BreadcrumbsBasic.twig (hunk 6 above)
Comment (pavelklibani) [reply packages/web-twig/src/Resources/components/Breadcrumbs/stories/BreadcrumbsBasic.twig]: empty line added

Comment (adamkudrna) [reply packages/web-twig/src/Resources/components/Breadcrumbs/stories/BreadcrumbsBasic.twig]: Hmm, this has already happened in another PR… @pavelklibani do you have Editorconfig installed in your IDE?

File: packages/web-react/src/components/Breadcrumbs/demo/BreadcrumbsCustom.tsx [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+      url: '#subcategoryUrl',
+    },
+    {
+      title: 'Current page',
+      url: '#currentUrl',
+    },
+  ];
+
+  return (
+    <Breadcrumbs goBackTitle="Back">
```
Comment (adamkudrna) [inline packages/web-react/src/components/Breadcrumbs/demo/BreadcrumbsCustom.tsx:26]: Icon separators seem to be missing from this example:

obrazek

Vs. `web`/`web-twig`:

obrazek

**Review by adamkudrna — APPROVED** [review]

File: packages/web-react/src/components/Breadcrumbs/demo/BreadcrumbsCustom.tsx (hunk 7 above)
Comment (pavelklibani) [reply packages/web-react/src/components/Breadcrumbs/demo/BreadcrumbsCustom.tsx:27]: Fixed, now it looks as it should

**Pull request merged**